MANUAL_INSTALL.md

```diff
@@ -0,0 +1,118 @@
+# Manual install
+
+## General
+
+HDM binds per default to port 3000.
+
+## Ruby
+
+Make sure you have the necessary ruby version on your system avaiable. You might use rvm or rbenv.
```

Comment:
maybe add a note that the minimal supported version is in .ruby-version file?

Reply:
added.